addons/mass_mailing_sms/wizard/mailing_sms_test_views.xml

```diff
@@ -6,10 +6,10 @@
         <field name="arch" type="xml">
             <form string="Send a Sample SMS">
                 <p class="text-muted">
-                    Send a sample SMS for testing purpose to the numbers below (comma-separated list).
+                    Send a sample SMS for testing purpose to the numbers below (carriage-return-separated list).
                 </p>
                 <group>
-                    <field name="numbers" placeholder="+32 495 85 85 77, +33 545 55 55 55"/>
+                    <field name="numbers" placeholder="+32 495 85 85 77&#10;+33 545 55 55 55"/>
```

Comment:
Same for e.g., both or none to update.

Reply:
Same answer, we tested various displays with LNA and this one was the voted best.
He is very picky on his placeholders, believe me...

Reply:
I believe you :D